packages/render-fidelity-tools/test/config.json

```diff
@@ -186,40 +153,72 @@
       }
     },
     {
-      "name": "khronos-NormalTangentMirrorTest",
-      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/NormalTangentMirrorTest/glTF/NormalTangentMirrorTest.gltf",
+      "name": "Astronaut",
```

Comment:
Let's remove the astronaut entirely; it's not even a khronos model and it doesn't test anything novel.